run-tests.js

```diff
@@ -8,7 +8,13 @@ var mocha = new Mocha()
 
 var testDir = 'test'
 require('babel/register')({
-  ignore: false
+  ignore: function (filename) {
+    if (filename.indexOf('weakee') > -1) {
+      return false
+    }
+
+    return filename.indexOf('node_modules') > -1
```

Comment:
Not having to publish a transpiled version of a module is one of the best things about JSPM. It's the feature that convinced me to use it instead of browserify or webpack. The problem with chai is unfortunate, but it seems to me this could be easily solved with a try{}catch(){} block, no?

Doesn't really matter, your solution is fine.